test/commands/shell.js

```diff
@@ -67,16 +78,18 @@ commands.kill = noop;
 commands.log = noop;
 commands.setprop = noop;
 
-module.exports = function shell(arg0) {
-  if (!arg0) {
+module.exports = function shell() {
+  var args = Array.prototype.slice.call(arguments).join(' ');
+
+  if (!args) {
     return;
   }
 
-  if (arg0.indexOf("'") === 0 && arg0.indexOf("'") === 0) {
-    arg0 = arg0.slice(1, -1);
+  if (args.indexOf("'") === 0 && args.indexOf("'") === 0) {
```

Comment:
wtf.

This came from my refactor to ES5 and I removed `startsWith` and `endsWith` and, well, fail.